muxink/src/codec/bincode.rs

```diff
@@ -45,7 +46,10 @@ where
     type Output = Bytes;
 
     fn transcode(&mut self, input: T) -> Result<Self::Output, Self::Error> {
-        bincode::serialize(&input).map(Bytes::from)
+        DefaultOptions::new()
```

Comment:
On second thought, I think it'd be good to simply hardcore all available options to save us grief later on, should they change.